portal-impl/src/com/liferay/portal/zip/ZipWriterFactoryImpl.java

```diff
@@ -43,4 +45,13 @@ public ZipWriter getZipWriter(File file) {
 		return new ZipWriterImpl(file);
 	}
 
+	@Override
```

Comment:
@holatuwol @hongvo2308 You also need the protection for PropsValues.ZIP_FILE_WRITER_EXPORT_BUFFER_SIZE here.
As an API method, you are not sure which zip writer will be used in export process.